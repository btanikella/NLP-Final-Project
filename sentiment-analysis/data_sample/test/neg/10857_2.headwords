juxtaposition
filmed
took
prepare
was
