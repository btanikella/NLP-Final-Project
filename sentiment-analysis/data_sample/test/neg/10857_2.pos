DT NN VBZ NN IN JJ JJ JJ JJ NN NN NNS  RB VBN CC IN DT JJ NN
DT IN DT NNS VBP RB VBN
IN PRP VBD DT JJ NN CC NN  CC VBD DT NN VBD DT NN WP VBD DT JJS NN IN DT NN CC VBD RP IN DT JJ CC JJ NN RB PRP MD VB JJ
IN PRP VBZ  VB IN DT JJ NN IN JJ NN NN NNS  DT IN WP VBP NN CC NN NNS IN DT NN DT NN CC VB NN IN DT VBG RB VBG JJ CC VBG PRP$ NN CC NNS JJ CC RB VBG TO VB IN DT NN CC NN  FW
EX VBD DT NN IN NN NN IN DT NNS NNS JJ NNS WDT VBD VBN CC DT NN WDT VBD RB VBN TO VB DT NN
