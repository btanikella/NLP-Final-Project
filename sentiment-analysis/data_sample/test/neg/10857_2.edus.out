Sentence #1 (21 tokens):
This movie is juxtaposition of various super bad tough guy biker characters, loosely connected but with no real storyline.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=9 CharacterOffsetEnd=14 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=15 CharacterOffsetEnd=17 PartOfSpeech=VBZ] [Text=juxtaposition CharacterOffsetBegin=18 CharacterOffsetEnd=31 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=32 CharacterOffsetEnd=34 PartOfSpeech=IN] [Text=various CharacterOffsetBegin=35 CharacterOffsetEnd=42 PartOfSpeech=JJ] [Text=super CharacterOffsetBegin=43 CharacterOffsetEnd=48 PartOfSpeech=JJ] [Text=bad CharacterOffsetBegin=49 CharacterOffsetEnd=52 PartOfSpeech=JJ] [Text=tough CharacterOffsetBegin=53 CharacterOffsetEnd=58 PartOfSpeech=JJ] [Text=guy CharacterOffsetBegin=59 CharacterOffsetEnd=62 PartOfSpeech=NN] [Text=biker CharacterOffsetBegin=63 CharacterOffsetEnd=68 PartOfSpeech=NN] [Text=characters CharacterOffsetBegin=69 CharacterOffsetEnd=79 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=79 CharacterOffsetEnd=80 PartOfSpeech=,] [Text=loosely CharacterOffsetBegin=81 CharacterOffsetEnd=88 PartOfSpeech=RB] [Text=connected CharacterOffsetBegin=89 CharacterOffsetEnd=98 PartOfSpeech=VBN] [Text=but CharacterOffsetBegin=99 CharacterOffsetEnd=102 PartOfSpeech=CC] [Text=with CharacterOffsetBegin=103 CharacterOffsetEnd=107 PartOfSpeech=IN] [Text=no CharacterOffsetBegin=108 CharacterOffsetEnd=110 PartOfSpeech=DT] [Text=real CharacterOffsetBegin=111 CharacterOffsetEnd=115 PartOfSpeech=JJ] [Text=storyline CharacterOffsetBegin=116 CharacterOffsetEnd=125 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=125 CharacterOffsetEnd=126 PartOfSpeech=.] 
root(ROOT-0, juxtaposition-4)
det(movie-2, This-1)
nsubj(juxtaposition-4, movie-2)
cop(juxtaposition-4, is-3)
prep(juxtaposition-4, of-5)
amod(characters-12, various-6)
amod(characters-12, super-7)
amod(characters-12, bad-8)
amod(characters-12, tough-9)
nn(characters-12, guy-10)
nn(characters-12, biker-11)
pobj(of-5, characters-12)
punct(juxtaposition-4, ,-13)
advmod(connected-15, loosely-14)
dep(juxtaposition-4, connected-15)
cc(connected-15, but-16)
conj(connected-15, with-17)
det(storyline-20, no-18)
amod(storyline-20, real-19)
pobj(with-17, storyline-20)
punct(juxtaposition-4, .-21)

Sentence #2 (8 tokens):
Some of the scenes are nicely filmed.
[Text=Some CharacterOffsetBegin=131 CharacterOffsetEnd=135 PartOfSpeech=DT] [Text=of CharacterOffsetBegin=136 CharacterOffsetEnd=138 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=139 CharacterOffsetEnd=142 PartOfSpeech=DT] [Text=scenes CharacterOffsetBegin=143 CharacterOffsetEnd=149 PartOfSpeech=NNS] [Text=are CharacterOffsetBegin=150 CharacterOffsetEnd=153 PartOfSpeech=VBP] [Text=nicely CharacterOffsetBegin=154 CharacterOffsetEnd=160 PartOfSpeech=RB] [Text=filmed CharacterOffsetBegin=161 CharacterOffsetEnd=167 PartOfSpeech=VBN] [Text=. CharacterOffsetBegin=167 CharacterOffsetEnd=168 PartOfSpeech=.] 
root(ROOT-0, filmed-7)
nsubj(filmed-7, Some-1)
prep(Some-1, of-2)
det(scenes-4, the-3)
pobj(of-2, scenes-4)
cop(filmed-7, are-5)
advmod(filmed-7, nicely-6)
punct(filmed-7, .-8)

Sentence #3 (39 tokens):
If they took the same cast and crew, and made a movie called the Gent who was the coolest character in the film and came up with an interesting and continuous story maybe it could be decent.
[Text=If CharacterOffsetBegin=173 CharacterOffsetEnd=175 PartOfSpeech=IN] [Text=they CharacterOffsetBegin=176 CharacterOffsetEnd=180 PartOfSpeech=PRP] [Text=took CharacterOffsetBegin=181 CharacterOffsetEnd=185 PartOfSpeech=VBD] [Text=the CharacterOffsetBegin=186 CharacterOffsetEnd=189 PartOfSpeech=DT] [Text=same CharacterOffsetBegin=190 CharacterOffsetEnd=194 PartOfSpeech=JJ] [Text=cast CharacterOffsetBegin=195 CharacterOffsetEnd=199 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=200 CharacterOffsetEnd=203 PartOfSpeech=CC] [Text=crew CharacterOffsetBegin=204 CharacterOffsetEnd=208 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=208 CharacterOffsetEnd=209 PartOfSpeech=,] [Text=and CharacterOffsetBegin=210 CharacterOffsetEnd=213 PartOfSpeech=CC] [Text=made CharacterOffsetBegin=214 CharacterOffsetEnd=218 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=219 CharacterOffsetEnd=220 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=221 CharacterOffsetEnd=226 PartOfSpeech=NN] [Text=called CharacterOffsetBegin=227 CharacterOffsetEnd=233 PartOfSpeech=VBD] [Text=the CharacterOffsetBegin=234 CharacterOffsetEnd=237 PartOfSpeech=DT] [Text=Gent CharacterOffsetBegin=238 CharacterOffsetEnd=242 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=243 CharacterOffsetEnd=246 PartOfSpeech=WP] [Text=was CharacterOffsetBegin=247 CharacterOffsetEnd=250 PartOfSpeech=VBD] [Text=the CharacterOffsetBegin=251 CharacterOffsetEnd=254 PartOfSpeech=DT] [Text=coolest CharacterOffsetBegin=255 CharacterOffsetEnd=262 PartOfSpeech=JJS] [Text=character CharacterOffsetBegin=263 CharacterOffsetEnd=272 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=273 CharacterOffsetEnd=275 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=276 CharacterOffsetEnd=279 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=280 CharacterOffsetEnd=284 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=285 CharacterOffsetEnd=288 PartOfSpeech=CC] [Text=came CharacterOffsetBegin=289 CharacterOffsetEnd=293 PartOfSpeech=VBD] [Text=up CharacterOffsetBegin=294 CharacterOffsetEnd=296 PartOfSpeech=RP] [Text=with CharacterOffsetBegin=297 CharacterOffsetEnd=301 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=302 CharacterOffsetEnd=304 PartOfSpeech=DT] [Text=interesting CharacterOffsetBegin=305 CharacterOffsetEnd=316 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=317 CharacterOffsetEnd=320 PartOfSpeech=CC] [Text=continuous CharacterOffsetBegin=321 CharacterOffsetEnd=331 PartOfSpeech=JJ] [Text=story CharacterOffsetBegin=332 CharacterOffsetEnd=337 PartOfSpeech=NN] [Text=maybe CharacterOffsetBegin=338 CharacterOffsetEnd=343 PartOfSpeech=RB] [Text=it CharacterOffsetBegin=344 CharacterOffsetEnd=346 PartOfSpeech=PRP] [Text=could CharacterOffsetBegin=347 CharacterOffsetEnd=352 PartOfSpeech=MD] [Text=be CharacterOffsetBegin=353 CharacterOffsetEnd=355 PartOfSpeech=VB] [Text=decent CharacterOffsetBegin=356 CharacterOffsetEnd=362 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=362 CharacterOffsetEnd=363 PartOfSpeech=.] 
root(ROOT-0, took-3)
mark(took-3, If-1)
nsubj(took-3, they-2)
det(cast-6, the-4)
amod(cast-6, same-5)
dobj(took-3, cast-6)
cc(cast-6, and-7)
conj(cast-6, crew-8)
punct(took-3, ,-9)
cc(took-3, and-10)
conj(took-3, made-11)
det(movie-13, a-12)
nsubj(called-14, movie-13)
dep(made-11, called-14)
det(Gent-16, the-15)
dobj(called-14, Gent-16)
nsubj(character-21, who-17)
cop(character-21, was-18)
det(character-21, the-19)
amod(character-21, coolest-20)
rcmod(Gent-16, character-21)
prep(character-21, in-22)
det(film-24, the-23)
pobj(in-22, film-24)
cc(called-14, and-25)
conj(called-14, came-26)
prt(came-26, up-27)
prep(came-26, with-28)
det(story-33, an-29)
amod(story-33, interesting-30)
cc(interesting-30, and-31)
conj(interesting-30, continuous-32)
pobj(with-28, story-33)
advmod(decent-38, maybe-34)
nsubj(decent-38, it-35)
aux(decent-38, could-36)
cop(decent-38, be-37)
amod(story-33, decent-38)
punct(took-3, .-39)

Sentence #4 (57 tokens):
As it is, prepare for a lame series of tough guy character intros, all of whom do nothing but ride bikes around the desert all day and cause trouble while all staying never getting sunburned and keeping their hair and clothes perfect and never needing to look for a home or money, etc.
[Text=As CharacterOffsetBegin=368 CharacterOffsetEnd=370 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=371 CharacterOffsetEnd=373 PartOfSpeech=PRP] [Text=is CharacterOffsetBegin=374 CharacterOffsetEnd=376 PartOfSpeech=VBZ] [Text=, CharacterOffsetBegin=376 CharacterOffsetEnd=377 PartOfSpeech=,] [Text=prepare CharacterOffsetBegin=378 CharacterOffsetEnd=385 PartOfSpeech=VB] [Text=for CharacterOffsetBegin=386 CharacterOffsetEnd=389 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=390 CharacterOffsetEnd=391 PartOfSpeech=DT] [Text=lame CharacterOffsetBegin=392 CharacterOffsetEnd=396 PartOfSpeech=JJ] [Text=series CharacterOffsetBegin=397 CharacterOffsetEnd=403 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=404 CharacterOffsetEnd=406 PartOfSpeech=IN] [Text=tough CharacterOffsetBegin=407 CharacterOffsetEnd=412 PartOfSpeech=JJ] [Text=guy CharacterOffsetBegin=413 CharacterOffsetEnd=416 PartOfSpeech=NN] [Text=character CharacterOffsetBegin=417 CharacterOffsetEnd=426 PartOfSpeech=NN] [Text=intros CharacterOffsetBegin=427 CharacterOffsetEnd=433 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=433 CharacterOffsetEnd=434 PartOfSpeech=,] [Text=all CharacterOffsetBegin=435 CharacterOffsetEnd=438 PartOfSpeech=DT] [Text=of CharacterOffsetBegin=439 CharacterOffsetEnd=441 PartOfSpeech=IN] [Text=whom CharacterOffsetBegin=442 CharacterOffsetEnd=446 PartOfSpeech=WP] [Text=do CharacterOffsetBegin=447 CharacterOffsetEnd=449 PartOfSpeech=VBP] [Text=nothing CharacterOffsetBegin=450 CharacterOffsetEnd=457 PartOfSpeech=NN] [Text=but CharacterOffsetBegin=458 CharacterOffsetEnd=461 PartOfSpeech=CC] [Text=ride CharacterOffsetBegin=462 CharacterOffsetEnd=466 PartOfSpeech=NN] [Text=bikes CharacterOffsetBegin=467 CharacterOffsetEnd=472 PartOfSpeech=NNS] [Text=around CharacterOffsetBegin=473 CharacterOffsetEnd=479 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=480 CharacterOffsetEnd=483 PartOfSpeech=DT] [Text=desert CharacterOffsetBegin=484 CharacterOffsetEnd=490 PartOfSpeech=NN] [Text=all CharacterOffsetBegin=491 CharacterOffsetEnd=494 PartOfSpeech=DT] [Text=day CharacterOffsetBegin=495 CharacterOffsetEnd=498 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=499 CharacterOffsetEnd=502 PartOfSpeech=CC] [Text=cause CharacterOffsetBegin=503 CharacterOffsetEnd=508 PartOfSpeech=VB] [Text=trouble CharacterOffsetBegin=509 CharacterOffsetEnd=516 PartOfSpeech=NN] [Text=while CharacterOffsetBegin=517 CharacterOffsetEnd=522 PartOfSpeech=IN] [Text=all CharacterOffsetBegin=523 CharacterOffsetEnd=526 PartOfSpeech=DT] [Text=staying CharacterOffsetBegin=527 CharacterOffsetEnd=534 PartOfSpeech=VBG] [Text=never CharacterOffsetBegin=535 CharacterOffsetEnd=540 PartOfSpeech=RB] [Text=getting CharacterOffsetBegin=541 CharacterOffsetEnd=548 PartOfSpeech=VBG] [Text=sunburned CharacterOffsetBegin=549 CharacterOffsetEnd=558 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=559 CharacterOffsetEnd=562 PartOfSpeech=CC] [Text=keeping CharacterOffsetBegin=563 CharacterOffsetEnd=570 PartOfSpeech=VBG] [Text=their CharacterOffsetBegin=571 CharacterOffsetEnd=576 PartOfSpeech=PRP$] [Text=hair CharacterOffsetBegin=577 CharacterOffsetEnd=581 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=582 CharacterOffsetEnd=585 PartOfSpeech=CC] [Text=clothes CharacterOffsetBegin=586 CharacterOffsetEnd=593 PartOfSpeech=NNS] [Text=perfect CharacterOffsetBegin=594 CharacterOffsetEnd=601 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=602 CharacterOffsetEnd=605 PartOfSpeech=CC] [Text=never CharacterOffsetBegin=606 CharacterOffsetEnd=611 PartOfSpeech=RB] [Text=needing CharacterOffsetBegin=612 CharacterOffsetEnd=619 PartOfSpeech=VBG] [Text=to CharacterOffsetBegin=620 CharacterOffsetEnd=622 PartOfSpeech=TO] [Text=look CharacterOffsetBegin=623 CharacterOffsetEnd=627 PartOfSpeech=VB] [Text=for CharacterOffsetBegin=628 CharacterOffsetEnd=631 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=632 CharacterOffsetEnd=633 PartOfSpeech=DT] [Text=home CharacterOffsetBegin=634 CharacterOffsetEnd=638 PartOfSpeech=NN] [Text=or CharacterOffsetBegin=639 CharacterOffsetEnd=641 PartOfSpeech=CC] [Text=money CharacterOffsetBegin=642 CharacterOffsetEnd=647 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=647 CharacterOffsetEnd=648 PartOfSpeech=,] [Text=etc. CharacterOffsetBegin=649 CharacterOffsetEnd=652 PartOfSpeech=FW] [Text=. CharacterOffsetBegin=652 CharacterOffsetEnd=653 PartOfSpeech=.] 
root(ROOT-0, prepare-5)
mark(is-3, As-1)
nsubj(is-3, it-2)
advcl(prepare-5, is-3)
punct(prepare-5, ,-4)
prep(prepare-5, for-6)
det(series-9, a-7)
amod(series-9, lame-8)
pobj(for-6, series-9)
prep(series-9, of-10)
amod(intros-14, tough-11)
nn(intros-14, guy-12)
nn(intros-14, character-13)
pobj(of-10, intros-14)
punct(intros-14, ,-15)
nsubj(do-19, all-16)
prep(all-16, of-17)
pobj(of-17, whom-18)
rcmod(intros-14, do-19)
dobj(do-19, nothing-20)
cc(do-19, but-21)
nn(bikes-23, ride-22)
conj(do-19, bikes-23)
prep(bikes-23, around-24)
det(desert-26, the-25)
pobj(around-24, desert-26)
det(day-28, all-27)
tmod(do-19, day-28)
cc(do-19, and-29)
conj(do-19, cause-30)
dobj(cause-30, trouble-31)
advmod(do-19, while-32)
pobj(while-32, all-33)
partmod(all-33, staying-34)
neg(getting-36, never-35)
xcomp(staying-34, getting-36)
dobj(getting-36, sunburned-37)
cc(sunburned-37, and-38)
conj(sunburned-37, keeping-39)
poss(hair-41, their-40)
nsubj(perfect-44, hair-41)
cc(hair-41, and-42)
conj(hair-41, clothes-43)
ccomp(keeping-39, perfect-44)
cc(perfect-44, and-45)
neg(needing-47, never-46)
conj(perfect-44, needing-47)
aux(look-49, to-48)
xcomp(needing-47, look-49)
prep(look-49, for-50)
det(home-52, a-51)
pobj(for-50, home-52)
cc(home-52, or-53)
conj(home-52, money-54)
punct(home-52, ,-55)
appos(home-52, etc.-56)
punct(prepare-5, .-57)

Sentence #5 (28 tokens):
There was a sort of background story about the characters pasts various people that were killed and a treasure that was probably supposed to be the plot.
[Text=There CharacterOffsetBegin=658 CharacterOffsetEnd=663 PartOfSpeech=EX] [Text=was CharacterOffsetBegin=664 CharacterOffsetEnd=667 PartOfSpeech=VBD] [Text=a CharacterOffsetBegin=668 CharacterOffsetEnd=669 PartOfSpeech=DT] [Text=sort CharacterOffsetBegin=670 CharacterOffsetEnd=674 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=675 CharacterOffsetEnd=677 PartOfSpeech=IN] [Text=background CharacterOffsetBegin=678 CharacterOffsetEnd=688 PartOfSpeech=NN] [Text=story CharacterOffsetBegin=689 CharacterOffsetEnd=694 PartOfSpeech=NN] [Text=about CharacterOffsetBegin=695 CharacterOffsetEnd=700 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=701 CharacterOffsetEnd=704 PartOfSpeech=DT] [Text=characters CharacterOffsetBegin=705 CharacterOffsetEnd=715 PartOfSpeech=NNS] [Text=pasts CharacterOffsetBegin=716 CharacterOffsetEnd=721 PartOfSpeech=NNS] [Text=various CharacterOffsetBegin=722 CharacterOffsetEnd=729 PartOfSpeech=JJ] [Text=people CharacterOffsetBegin=730 CharacterOffsetEnd=736 PartOfSpeech=NNS] [Text=that CharacterOffsetBegin=737 CharacterOffsetEnd=741 PartOfSpeech=WDT] [Text=were CharacterOffsetBegin=742 CharacterOffsetEnd=746 PartOfSpeech=VBD] [Text=killed CharacterOffsetBegin=747 CharacterOffsetEnd=753 PartOfSpeech=VBN] [Text=and CharacterOffsetBegin=754 CharacterOffsetEnd=757 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=758 CharacterOffsetEnd=759 PartOfSpeech=DT] [Text=treasure CharacterOffsetBegin=760 CharacterOffsetEnd=768 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=769 CharacterOffsetEnd=773 PartOfSpeech=WDT] [Text=was CharacterOffsetBegin=774 CharacterOffsetEnd=777 PartOfSpeech=VBD] [Text=probably CharacterOffsetBegin=778 CharacterOffsetEnd=786 PartOfSpeech=RB] [Text=supposed CharacterOffsetBegin=787 CharacterOffsetEnd=795 PartOfSpeech=VBN] [Text=to CharacterOffsetBegin=796 CharacterOffsetEnd=798 PartOfSpeech=TO] [Text=be CharacterOffsetBegin=799 CharacterOffsetEnd=801 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=802 CharacterOffsetEnd=805 PartOfSpeech=DT] [Text=plot CharacterOffsetBegin=806 CharacterOffsetEnd=810 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=810 CharacterOffsetEnd=811 PartOfSpeech=.] 
root(ROOT-0, was-2)
expl(was-2, There-1)
det(sort-4, a-3)
nsubj(was-2, sort-4)
prep(sort-4, of-5)
nn(story-7, background-6)
pobj(of-5, story-7)
prep(story-7, about-8)
det(characters-10, the-9)
nsubj(pasts-11, characters-10)
pcomp(about-8, pasts-11)
amod(people-13, various-12)
dobj(pasts-11, people-13)
nsubjpass(killed-16, that-14)
auxpass(killed-16, were-15)
rcmod(people-13, killed-16)
cc(killed-16, and-17)
det(treasure-19, a-18)
conj(killed-16, treasure-19)
nsubjpass(supposed-23, that-20)
auxpass(supposed-23, was-21)
advmod(supposed-23, probably-22)
rcmod(treasure-19, supposed-23)
aux(plot-27, to-24)
cop(plot-27, be-25)
det(plot-27, the-26)
xcomp(supposed-23, plot-27)
punct(was-2, .-28)

